src/main/java/org/killbill/billing/payment/api/PaymentAttempt.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.payment.api;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+import org.joda.time.DateTime;
+import org.killbill.billing.catalog.api.Currency;
+import org.killbill.billing.util.entity.Entity;
+
+public interface PaymentAttempt extends Entity {
+
+    /**
+     * @return the id of the payment.
+     */
+    UUID getPaymentId();
+
+    /**
+     * @return the type of transaction
+     */
+    TransactionType getTransactionType();
+
+    /**
+     * @return the amount
+     */
+    BigDecimal getAmount();
+
+    /**
+     * @return the currency associated with that payment
+     */
+    Currency getCurrency();
+
+    /**
+     * @return the status for that transaction
+     */
+    TransactionStatus getTransactionStatus();
```

Comment:
@pierre 
So, do you mean I would need to use [these](https://github.com/javier-gomez-hs/killbill/blob/b4787d47160d1257697757e89188b09d8664c411/payment/src/main/java/org/killbill/billing/payment/dao/PaymentAttemptModelDao.java#L41-L51) parameters as part of my new [DefaultPaymentAttempt](https://github.com/javier-gomez-hs/killbill/blob/b657099c18e3da02216f4e17be86fa18b86015ce/payment/src/main/java/org/killbill/billing/payment/api/DefaultPaymentAttempt.java)? (with the specific get Methods in the original Interface in **killbill-api**) 
Sorry but I don't understand what do you mean with "expose the attributes". 

Thank you!